plugin/trino-resource-group-managers/src/main/resources/db/migration/oracle/V1__add_resource_groups_schema.sql

```diff
@@ -0,0 +1,44 @@
+CREATE TABLE resource_groups_global_properties (
+    name VARCHAR(128) NOT NULL PRIMARY KEY,
+    value VARCHAR(512) NULL,
+    CHECK (name in ('cpu_quota_period'))
+);
+
+CREATE TABLE resource_groups (
```

Comment:
Note that in Oracle, the database issues an implicit COMMIT statement before and after every DDL statement.  That means this migration is not transactional.  We should break it up into multiple files.

Reply:
@aalbu thanks! I wasn't aware Oracle did not have transactional DDL. Updated to break the migration into multiple files.

Reply:
Anything other than PostgreSQL doesn't have transactional DDL IIRC, please check if this change also needs to be done for MySQL.

Reply:
@hashhar I already have it in place for MySQL. The only migration I have now that does all DDL in 1 migration is PostgreSQL.